### PR TITLE
Add step component

### PR DESCRIPTION
**Pull request by hong-duc**

### CODE REVIEW

File: docs/examples/elements/StepExample/Attached.example.vue [hunk 1]
```diff
@@ -1,6 +1,25 @@
 <template lang="html">
   <div>
-    <sui-step-group attached="top">
+    <sui-step-group step-number="three" attached="top">
```
Comment (hong-duc) [inline docs/examples/elements/StepExample/Attached.example.vue]: should I pass number then map to string, or just pass string like this ?

**Comment by hong-duc** [comment]
I have added all example, the only thing left is the review

File: docs/examples/elements/StepExample/Attached.example.vue (hunk 1 above)
Comment (mariolamacchia) [reply docs/examples/elements/StepExample/Attached.example.vue]: I'd say we should support both. We already have a util function and prop type:
https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/992d1ffdbc691eb9a56074820fc121081bbd14b8/src/lib/mixins/SemanticUIVueMixin/utils.js#L7
https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/992d1ffdbc691eb9a56074820fc121081bbd14b8/src/lib/PropTypes.js#L47

Comment (hong-duc) [reply docs/examples/elements/StepExample/Attached.example.vue]: oh so you already have util func for it, thank you

**Comment by MessineseGianluca** [comment]
Could you edit your yarn.lock as the current one in the master? Thank you.

**Comment by hong-duc** [comment]
@MessineseGianluca  I have merged with your master branch

**Comment by mariolamacchia** [comment]
Looks great, thanks!

**Pull request merged**